### ISSUE TITLE
App: Climate control: "Timer" page is blank

### ISSUE THREAD

**Issue by cloveras** [electrichasgoneaudi/q6-e-tron#18]
When navigating from the home page to "Climate control" and then to "Timer", the app only displays this text: 

"Please specify the time at which your vehicle should have reached the preset temperature."

The rest of the page is blank so it's impossible to set the timer.

![IMG_2147](https://github.com/user-attachments/assets/123cb9b2-bed5-4b7e-bcb5-8f2d540b4fe3)

**Comment by bwa** [comment]
@cloveras Have you discovered that the timers need to be created in Car's MMI, then it will appear in app. 

This is in my App after creating timers in MMI

![image](https://github.com/user-attachments/assets/0041f91b-cb9a-465f-92e8-0795c299de11)

**Comment by cloveras** [comment]
Aha! Thanks.

But I still think this is a bug in the app. It makes no sense to say "Please specify the time [...]" when it's not possible to specify anything. At the very minimum there should be some information about doing it in the car.

I don't think anyone in Audi think the current functionality in the app is good enough for a "premium" brand.

**Comment by bwa** [comment]
I completely agree. But the bug is in my Audi App, and the issue decription should maybe change to be more clear about that.

**Comment by bwa** [comment]
Fixed in MyAudi App 5.x
